pyfr/plugins/tavg.py

```diff
@@ -63,18 +69,30 @@ def _prepare_exprs(self):
         c = self.cfg.items_as('constants', float)
         self.anames, self.aexprs = [], []
         self.outfields, self.fexprs = [], []
-
+        self.vnames, self.fnames = [], []
+        
         # Iterate over accumulation expressions first
         for k in cfg.items(cfgsect, prefix='avg-'):
-            self.anames.append(k.removeprefix('avg-'))
-            self.aexprs.append(cfg.getexpr(cfgsect, k, subs=c))
-            self.outfields.append(k)
+                self.anames.append(k.removeprefix('avg-'))
+                self.aexprs.append(cfg.getexpr(cfgsect, k, subs=c))
+                self.outfields.append(k)
 
         # Followed by any functional expressions
         for k in cfg.items(cfgsect, prefix='fun-avg-'):
-            self.fexprs.append(cfg.getexpr(cfgsect, k, subs=c))
-            self.outfields.append(k)
-
+                self.fnames.append(k.removeprefix('fun-avg-'))
+                self.fexprs.append(cfg.getexpr(cfgsect, k, subs=c))
+                self.outfields.append(k)
+
+        # Create fields for std deviations        
+        if self.dev_mode == 'all':
+            for k in cfg.items(cfgsect, prefix='avg-'):
+                    self.outfields.append(re.sub('avg-', 'std-dev-', k))
```

Comment:
Convention from NumPy and Spreadsheets is to abbreviate as std or stdev.  Also, no need for a regular expression here: just `f'stdev-{k[4:]}'`